bioimageio/spec/shared/fields.py

```diff
@@ -169,6 +167,14 @@ def __init__(self, *super_args, **super_kwargs):
         super().__init__(*super_args, **super_kwargs)
         self.type_name += f"\\[{' | '.join(cf.type_name for cf in self._candidate_fields)}\\]"  # add types of options
 
+    def _deserialize(self, value, attr=None, data=None, **kwargs):
+        try:
+            return super()._deserialize(value, attr=attr, data=data, **kwargs)
+        except ValidationError as e:
+            errors = sorted(e.messages, key=lambda msg: len(msg))
+            messages = ["Errors in all options for this field. Fix any of the following errors:"] + errors
```

Comment:
Does this parse well? If I try this the output is just a single list without new lines:
```
raise ValidationError(message=["a", "b", "c"])
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
<ipython-input-3-e0174c71d04f> in <module>
----> 1 raise ValidationError(message=["a", "b", "c"])

ValidationError: ['a', 'b', 'c']
```

Reply:
none of the error messages that are returned anywhere are followed by new lines. This formatting is left to the code displaying the error message. When using the `validate` command pretty pprint is used to display the nested errors. I think we should leave the formatting there